plugins/karma.js

```diff
@@ -0,0 +1,87 @@
+const RTM_EVENTS = require('@slack/client').RTM_EVENTS;
+const storage = require('node-persist');
+const winston = require('winston');
+
+const META = {
+    name: 'karma',
+    short: 'Give someone karma points.',
+    examples: [
+        'phr34ck +5 @bassem',
+        '@bosta karma @phr34ck',
```

Comment:
Maybe we should change that to the same user that has been granted the karma so that the slow ones don't get confused :D